src/App.tsx

```diff
@@ -1,43 +1,33 @@
-import { Tldraw, useFileSystem } from "@tldraw/tldraw";
-import { useUsers } from "y-presence";
-import { useMultiplayerState } from "./hooks/useMultiplayerState";
-import "./index.css";
-import { awareness, roomID } from "./store/store";
+import { Tldraw, useFileSystem } from '@tldraw/tldraw';
+import { useUsers } from 'y-presence';
+import { useMultiplayerState } from './hooks/useMultiplayerState';
+import './index.css';
```

Comment:
Maybe we could move the `App.tsx`-related CSS to `App.css` and try to stick to the rule one `.tsx` file - one `.css` file, what do you think? BTW I would even suggest using CSS-modules for the React components in the future.